src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -502,10 +502,6 @@ extern "C" int32_t CryptoNative_SetEncryptionPolicy(SSL_CTX* ctx, EncryptionPoli
     return SSL_CTX_set_cipher_list(ctx, cipherString);
 }
 
-extern "C" void CryptoNative_SslCtxSetClientCAList(SSL_CTX* ctx, X509NameStack* list)
-{
-    SSL_CTX_set_client_CA_list(ctx, list);
-}
```

Comment:
I think a reference of ```SSL_CTX_set_client_CA_list``` can also be found in src/Native/Unix/System.Security.Cryptography.Native/opensslshim.h. Can you do a string search in whole of repo of removed functions, and the functions they call in turn etc for cleanup.

Reply:
Okay, also `DuplicateX509Name` is no longer used, will remove it as well